web/src/lib/components/assets/thumbnail/image-thumbnail.svelte

```diff
@@ -12,10 +13,19 @@
   export let curve = false;
   export let shadow = false;
   export let circle = false;
+  export let displayPanoramaArrow = false;
 
   let complete = false;
 </script>
 
+{#if displayPanoramaArrow}
+  <div class="absolute right-0 top-0 text-white text-xs font-medium flex gap-1 place-items-center z-20">
+    <span class="pt-2 pr-2">
+      <Rotate360Icon size="24" />
+    </span>
+  </div>
+{/if}
+
```

Comment:
I don't think this is the correct place for this icon. We have other icons on thumbnails to indicate if it is archived, favorited, etc. Those live in `web/src/lib/components/assets/thumbnail/thumbnail.svelte`. I believe that would be the appropriate place for this as well. It also removes the need for `displayPanoramaArrow` in this component.

Reply:
I moved it to thumbnail.svelte because panorama icon can be potentially displayed for videos as well. 

But as far as I see here video-related icons are rendered in video-thumbnail component
https://github.com/immich-app/immich/blob/8a3ab5be3e2628a5c777876eab3dc23ac519f2e6/web/src/lib/components/assets/thumbnail/video-thumbnail.svelte

Reply:
Yeah, we can cross that bridge when we get there. I think this is the right place for now.